NuRadioMC/SignalGen/parametrizations.py

```diff
@@ -21,13 +21,16 @@ def set_log_level(level):
 
 """
 
+_random_generators = {}
+_Alvarez2009_k_L = None
+
 
 def get_parametrizations():
     """ returns a list of all implemented parametrizations """
     return ['ZHS1992', 'Alvarez2000', 'Alvarez2009', 'Alvarez2012']
 
 
-def get_time_trace(energy, theta, N, dt, shower_type, n_index, R, model):
+def get_time_trace(energy, theta, N, dt, shower_type, n_index, R, model, seed=None, same_shower=False):
```

Comment:
Could you add same_shower to the docstring?

Reply:
done